templates/files/k8s-resource/calico-all.yaml

```diff
@@ -1046,6 +1069,13 @@ spec:
               name: cni-net-dir
             - mountPath: /calico-secrets
               name: etcd-certs
+        # Adds a Flex Volume Driver that creates a per-pod Unix Domain Socket to allow Dikastes
+        # to communicate with Felix over the Policy Sync API.
+        - name: flexvol-driver
+          image: {{.DockerRegistry}}/giantswarm/pod2daemon-flexvol:v3.9.3
+          volumeMounts:
+            - name: flexvol-driver-host
+              mountPath: /host/driver
```

Comment:
What is the intention behind of using pod2daemon-flexvol? 
In the comment say to allow Dikastes communicate with Felix but I don´t see dikastes images.

If it is all intended, it could be great to add some info or link in the PR description about new things to understand what are we changing

Reply:
Honestly I have no idea.
The pod and the comment come from the official .yaml: https://docs.projectcalico.org/v3.9/manifests/calico-etcd.yaml

Reply:
I think this should be checked and documented/communicated. It's not fully clear what this or Dikastes does, but what I can find from scarce resources in Calico documentation and Google searches, it seems to be a side-car container used with Envoy in Istio setups and potentially related to network policy configurations. I'm not sure if we should have something like this enabled out of the box, but I'm also wondering if this is requirement when customer wants to run Istio on top of our clusters.

Ping @pipo02mix - do you potentially know more about this wrt. Istio?

Reply:
Here is well explained https://github.com/projectcalico/app-policy

So it is not only for IStio (you can use to add policies at layer 7 for your apps) but Istio makes it easier to use. I would like to have it, but I would do it in a different story (PR)